auction_keeper/main.py

```diff
@@ -104,12 +102,21 @@ def __init__(self, args: list, **kwargs):
         parser.add_argument("--model", type=str, required=True, nargs='+',
                             help="Commandline to use in order to start the bidding model")
 
-        parser.add_argument("--ethgasstation-api-key", type=str, default=None, help="ethgasstation API key")
-        parser.add_argument('--etherchain-gas-price', dest='etherchain_gas', action='store_true',
-                            help="Use etherchain.org gas price")
-        parser.add_argument('--poanetwork-gas-price', dest='poanetwork_gas', action='store_true',
-                            help="Use POANetwork gas price")
+        gas_group = parser.add_mutually_exclusive_group()
+        gas_group.add_argument("--ethgasstation-api-key", type=str, default=None, help="ethgasstation API key")
+        gas_group.add_argument('--etherchain-gas-price', dest='etherchain_gas', action='store_true',
+                               help="Use etherchain.org gas price")
+        gas_group.add_argument('--poanetwork-gas-price', dest='poanetwork_gas', action='store_true',
+                               help="Use POANetwork gas price")
+        gas_group.add_argument('--fixed-gas-price', type=float, default=None,
+                               help="Uses a fixed value (in Gwei) instead of an external API to determine initial gas")
         parser.add_argument("--poanetwork-url", type=str, default=None, help="Alternative POANetwork URL")
+        parser.add_argument("--gas-initial-multiplier", type=float, default=1.0,
+                            help="Adjusts the initial API-provided 'fast' gas price, default 1.0")
+        parser.add_argument("--gas-reactive-multiplier", type=float, default=2.25,
+                            help="Increases gas price when transactions haven't been mined after some time")
+        parser.add_argument("--gas-maximum", type=float, default=100000,
```

Comment:
This is, admittedly, quite aggressive.  I figured it better to have an aggressive maximum rather than let all the default keepers fail amidst congestion.